src/db/product.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "name": "baseball",
```

Comment:
Your product list is missing the id attribute. However, your router has get product by id.